app/src/main/java/com/example/campusguide/directions/indoor/Graph.kt

```diff
@@ -13,13 +13,13 @@ class Graph constructor(building: Building) {
     init {
         // Add all rooms to the map
         building.rooms.forEach { room ->
-            val floor = room.code[0].toInt()
+            val floor = room.code.substring(0, 1).toInt()
```

Comment:
Can you explain what's happening here?

Reply:
This is to fix a bug that went unnoticed before because floor numbers weren't actually used
Red line = turns a character into an int, which gives you the char code instead of the number it represents. So "200.01" is considered to be on floor 50 since the char code for 2 is 50
Green line = turns the string into an int, which works properly. So "200.01" is considered to be on floor 2